app/models/manageiq/providers/oracle_cloud/cloud_manager/metrics_capture.rb

```diff
@@ -0,0 +1,119 @@
+class ManageIQ::Providers::OracleCloud::CloudManager::MetricsCapture < ManageIQ::Providers::CloudManager::MetricsCapture
+  delegate :ext_management_system, :to => :target
+
+  VIM_STYLE_COUNTERS = {
```

Comment:
Note to self: I wonder if these could live in the BaseManager::MetricsCapture class, for every provider except VMware I believe these are hard-coded